### DISCUSSION TITLE
What is an Information Flow? never marks as completed

### DISCUSSION THREAD

**Discussion by johanlantz** [OpenMined/courses#205] [Course: Our Privacy Opportunity] [answered]
Hi, I am stuck at the "What is an Information Flow?" module. I have let it run to the end several times but it is never marked as completed so I can't move on to the next module.

Using Chrome browser on a Macbook

**Comment by dhreb** [comment, marked as answer]
Hi @johanlantz , on several of the concepts, there are little quizzes below the videos. Try scrolling down and see if there is a quiz there to complete.

**Comment by johanlantz** [comment]
@bottomparm ouch, that was embarrassing. When I did the lesson I was not in full-screen mode and no external monitor so the video just fit my screen and I did not consier there could be anything below. 

Today when starting up things I had the external monitor connected and then the quiz was naturally visible :-) I should have just scrolled down. 

Thanks for the pointer 😊
